client/packages/system/src/Location/Components/LocationSearchInput.tsx

```diff
@@ -21,11 +21,15 @@ export const LocationSearchInput: FC<LocationSearchInputProps> = ({
   disabled,
   autoFocus = false,
 }) => {
-  const { data, isLoading } = useLocation.document.listAll({
+  const { mutateAsync, data, isLoading } = useLocation.document.listAll({
     direction: 'asc',
     key: 'name',
   });
 
+  useEffect(() => {
```

Comment:
this is the location input fix. An alternative is to change the hook to use `useQuery` but I like that the csv export only queries when the button is clicked, so I've gone with the mutation

Reply:
I was going to suggest some sort of wrapper around useQuery that allows `easy` to consume way to do the same as the useMutation but with useQuery, but then realised it might be an overkill, this is fine afaik just a little bit confusing, can suggest one thing, in `useLocationAll.ts` (and maybe for others like this that need to fetched later)   `return { ...result, fetchAsync: result.mutateAsync };`, then it's just a little bit clearer.

Reply:
thanks, have done - it's much clearer calling `fetchAsync` than it is `mutateAsync`!